src/softmmu_template.h

```diff
@@ -160,17 +160,20 @@ DATA_TYPE glue(glue(io_read, SUFFIX), MMUSUFFIX)(CPUArchState *env, target_phys_
 
     SE_SET_MEM_IO_VADDR(env, addr, 0);
 
+    if (likely(!g_sqi.mem.is_mmio_symbolic(addr, DATA_SIZE))) {
```

Comment:
Actually, you would still need to call the original handler for concolic execution (you need the original concrete value).

Reply:
This  commit could be sent to a separate PR